tests/e2e_tests/common/user_steps.py

```diff
@@ -1,8 +1,9 @@
 from iotqautils.gtwRest import Rest_Utils_SBC

```

Comment:
Split this files in two:
- one for common steps for your own repository
- general purpose library with iotagent/iotmanager API